docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md

```diff
@@ -78,7 +78,23 @@ Installing YugabyteDB Anywhere on airgapped hosts, without access to any Interne
   `https://downloads.yugabyte.com`
```

Comment:
Remove the "Installing on airgapped hosts requires additional configurations..." sentence above, the section is right here.
Change first sentence "Installing YugabyteDB Anywhere on airgapped hosts (without access to any Internet traffic (inbound or outbound)) requires the following additional configuration:"